jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/TargetSystemIntegrationTest.java

```diff
@@ -167,7 +167,7 @@ protected final void waitAndAssertMetrics(Consumer<Metric>... assertions) {
   protected void verifyMetrics() {
     MetricsVerifier metricsVerifier = createMetricsVerifier();
     await()
-        .atMost(Duration.ofSeconds(30))
+        .atMost(Duration.ofSeconds(60))
```

Comment:
Increase maximum assertions timeout to give tested target more time to stabilize and generate all the metrics